tests/test_ops.py

```diff
@@ -1239,28 +1240,44 @@ def test_mathtransform():
 
+         #{"opName": "adjust_contrast", "outName": "adjust_contrast/rank3_float32", "varShapes": [[8,8,3], []],"varTypes": ["float32", "float32"], "varInit": ["uniform", "uniform", "two"]},
+         #{"opName": "adjust_contrast", "outName": "adjust_contrast_v/rank3_mixed", "varShapes": [[8,8,3,1], []],"varTypes": ["float32", "float32"], "varInit": ["uniform", "two"]},
+         #{"opName": "adjust_contrast", "outName": "adjust_contrast_v/rank4_mixed","varShapes": [[8, 8, 3, 4], []], "varTypes": ["float32", "float32"], "varInit": ["uniform", "two"]},
+
+        #{"opName": "crop_and_resize", "outName": "crop_and_resize", "varShapes": [[1,2,2,1], [2,4], [2], [2]],"varTypes": ["float32", "float32","int32","int32"], "varInit": ["uniform", "uniform","uniform_int10","uniform_int10"]},
+
+
+        # {"opName": "draw_bounding_boxes", "outName": "draw_bounding_boxes", "varShapes": [[2,256,256,1], [2,2,4]],"varTypes": ["float32", "float32"], "varInit": ["uniform", "uniform"]},
+
+        # {"opName": "resize_bilinear", "outName": "resize_bilinear/float32", "varShapes": [[2,256,256,1], [2]],"varTypes": ["float32", "int32"], "varInit": ["uniform", "uniform_int2"]},
+        # {"opName": "resize_bilinear", "outName": "resize_bilinear/float64", "varShapes": [[2, 256, 256, 1], [2]],  "varTypes": ["float64", "int32"], "varInit": ["uniform", "uniform_int2"]},
+
+        # {"opName": "resize_nearest_neighbor", "outName": "resize_nearest_neighbor/float32", "varShapes": [[2,256,256,1], [2]],"varTypes": ["float32", "int32"], "varInit": ["uniform", "uniform_int10"]},
+        #  {"opName": "resize_nearest_neighbor", "outName": "resize_nearest_neighbor/float64", "varShapes": [[2,256,256,1], [2]],"varTypes": ["float64", "int32"], "varInit": ["uniform", "uniform_int10"]},
 
         # {"opName": "check_numerics", "outName": "check_numerics/rank1_float16", "varShapes":[[5]], "varTypes":["float16", "string"], "varInit":["uniform", "string_scalar"], "message":"This is a test string."},
         # {"opName": "check_numerics", "outName": "check_numerics/rank1_float32", "varShapes":[[5]], "varTypes":["float32", "string"], "varInit":["uniform", "string_scalar"], "message":"This is a test string."},
         # {"opName": "check_numerics", "outName": "check_numerics/rank2_float64", "varShapes":[[5]], "varTypes":["float64", "string"], "varInit":["uniform", "string_scalar"], "message":"This is a test string."},
         # {"opName": "check_numerics", "outName": "check_numerics/rank4_bfloat16", "varShapes":[[5]], "varTypes":["bfloat16", "string"], "varInit":["uniform", "string_scalar"], "message":"This is a test string."}
 
+        # {"opName": "dropout", "outName": "dropout", "varShapes":[[1,100]], "varTypes":["float32"], "varInit":["uniform"]}
```

Comment:
One problem with testing dropout: no way to validate results, due to randomness...
And I think this is implemented as multiple ops internally, not one op.
So not a high priority to test...